src/panels/lovelace/cards/hui-picture-glance-card.ts

```diff
@@ -189,8 +192,10 @@ class HuiPictureGlanceCard extends LitElement implements LovelaceCard {
 
     return html`
       <ha-icon
-        .entity="${stateObj.entity_id}"
-        @click="${dialog ? this._openDialog : this._callService}"
+        @ha-click=${this._handleTap}
+        @ha-hold=${this._handleHold}
+        .longPress=${longPress()}
+        .config=${entityConf}
```

Comment:
No need for this, can use `this._config` in the event handler.

Reply:
This is the config for just this entity. We don't actions for the parent

Reply:
Ah I see :-)